udata/core/organization/api.py

```diff
@@ -87,8 +87,13 @@ def get(self, org):
     @api.expect(org_fields)
     @api.marshal_with(org_fields)
     @api.response(400, errors.VALIDATION_ERROR)
+    @api.response(410, 'Organization has been deleted')
     def put(self, org):
-        '''Update a organization given its identifier'''
+        '''
+        Update a organization given its identifier
+
+        :raises PermissionDenied:
+        '''
         if org.deleted:
             api.abort(410, 'Organization has been deleted')
```

Comment:
The decorator is only here to expose the documentation which can't be extracted.